Partition (WBP)/Functions/VI0049C.VTI0049.fun.yml

```diff
@@ -89,8 +100,85 @@
     Endif
     Endroutine
     
+    *
+    * Execute Sql Script on MSSQL, MySql,Azure SQL, Oracle and SQL Anywhere
+    *
+    Mthroutine Name(mthExecuteSqlScript)
+    Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
+    #LF_rc := OK
+    #LF_logfile_name := *TEMP_DIR + 'PerformanceMetricsLog.txt'
+    
+    Case Of_Field(#LF_DB_Type)
+    When Value_Is(= MSSQLS)
+    * #LF_commandLine_script := 'SQLCMD.exe -S Localhost -U Lansa -P Pcxuser122robg2 -i ' + *ROOT_DIR + '\Lansa\LANSA\VersionControl\Scripts\mssql-performance-statistics-bind.sql'
+    #LF_commandLine_script := *ROOT_DIR + '\Lansa\LANSA\VersionControl\Scripts\new.cmd'
+    When Value_Is(= SQLAZURE)
+    #LF_commandLine_script := ''
+    
+    When Value_Is(= ODBCORACLE)
+    #LF_commandLine_script := ''
+    
+    When Value_Is(= MYSQL)
+    #LF_commandLine_script := 'mysql --host="mysql150060.cnyed5gpqwey.us-east-1.rds.amazonaws.com" --database="LANSA" --user="lansa" --password="Pcxuser122robg2" < ' + *ROOT_DIR + '\MySQL\LANSA\VersionControl\Scripts\mysql-performance-statistics-bind.sql'
+    
+    When Value_Is(= SQLANYWHERE)
+    #LF_commandLine_script := ''
+    Endcase
+    
+    * Non-working
+    * Use Builtin(SYSTEM_COMMAND) With_Args('X' #LF_commandLine_script ' ' #LF_logfile_name) To_Get(#STD_NUM)
+    * Use Builtin(SYSTEM_COMMAND) With_Args(B (' #LF_commandLine_script >> #LF_logfile_name 2>&1')) To_Get(#STD_NUM)
+    
+    * Use Builtin(SYSTEM_COMMAND) With_Args('S' '#LF_commandLine_script > #LF_logfile_name') To_Get(#STD_NUM)
+    * Use Builtin(SYSTEM_COMMAND) With_Args(B ('C:\Users\Administrator.EC2AMAZ-0G5MVIP\Desktop\PRDDocs\new.sql >> C:\Users\Administrator.EC2AMAZ-0G5MVIP\Desktop\PRDDocs\output1.txt 2>&1')) To_Get(#STD_NUM)
+    
+    * Working Command
```

Comment:
If its working why is it commented out?

Reply:
It is because both the input command and redirected output text file are absolute path. [Hardcoded]. I validated the log written into the output file.